contracts/WooCrossChainRouter.sol

```diff
@@ -278,10 +252,40 @@ contract WooCrossChainRouterV2 is IWooCrossChainRouterV2, Ownable, ReentrancyGua
                     bridgedToken,
                     amountLD,
                     toToken,
-                    bridgedToken,
+                    toToken,
                     minToAmount,
                     amountLD
                 );
+            } else {
+                TransferHelper.safeApprove(bridgedToken, address(wooRouter), amountLD);
+                try wooRouter.swap(bridgedToken, toToken, amountLD, minToAmount, payable(to), to) returns (
+                    uint256 realToAmount
```

Comment:
sorry to bother, I didn't remember very clearly . Want to double check:

   realToAmount can be captured as the return value?